tests/integration/components/frost-checkbox-test.js

```diff
@@ -140,20 +139,24 @@ describeComponent(
     })
 
     it('triggers value change', function () {
-      this.set('checkbox-value', '')
-      this.on('valueChange', function (val) {
-        this.set('checkbox-value', val)
+      this.set('checkbox-value', false)
+      this.on('valueChange', function (attrs) {
+        this.set('checkbox-value', attrs.value)
       })
-      this.render(hbs`{{#frost-checkbox
+      this.render(hbs`{{frost-checkbox
         id="value"
         value="value"
-        on-input=(action "valueChange")}}value{{/frost-checkbox}}
+        onInput=(action 'valueChange')
```

Comment:
Interesting, I probably would have just passed in a stub. 

```js
let inputHandler
...
beforeEach(function () {
  ...
  inputHandler = sandbox.stub()
  this.setProperties({inputHandler})
})

...

this.render(hbs`
  {{frost-checkbox
    id='value'
    label='value'
    onInput=inputHandler
    value='value'
  }}
`)

...

expect(inputHandler).to.have.been.calledWith(sinon.match({value: 'value'}))
```

Not saying you should change it, just pointing out another way to do it as well, if you're a `sinon` lover (like myself).